projects/BFDR/BirthRecord_submissionProperties.cs

```diff
@@ -3140,5 +3140,265 @@ public string FatherEthnicityLiteral
 
             }
         }
+
+/// TODO: Required field in FHIR, needs BLANK placeholder
+        /// <summary>Family name of attendant.</summary>
+        /// <value>the attendant's family name (i.e. last name)</value>
+        /// <example>
+        /// <para>// Setter:</para>
+        /// <para>ExampleBirthRecord.AttendantFamilyName = "Seito";</para>
+        /// <para>// Getter:</para>
+        /// <para>Console.WriteLine($"Attendants's Name: {ExampleBirthRecord.AttendantFamilyName}");</para>
+        /// </example>
+        [Property("Attendant Name", Property.Types.String, "Birth Certification", "Family name of attendant.", true, VR.IGURL.Practitioner, true, 6)]
+        [FHIRPath("Bundle.entry.resource.where($this is Practitioner)", "name")]
+        public string AttendantFamilyName
```

Comment:
I remember talking about this field being required, but now looking at the IG, I don't think family name is https://hl7.org/fhir/R4/datatypes.html#HumanName

Reply:
looking at our IG, under the Key Elements tab, it says the cardinality for name.family is 1..1?

https://hl7.org/fhir/us/vr-common-library/2024Jan/StructureDefinition-Practitioner-vr.html